django_loci/channels/asgi.py

```diff
@@ -6,15 +6,15 @@
 from django_loci.channels.base import location_broadcast_path
 from django_loci.channels.consumers import LocationBroadcast
 
-channel_routing = ProtocolTypeRouter(
+application = ProtocolTypeRouter(
```

Comment:
LGTM, except `application` is a very generic variable name, if any library in any of the dependencies use this name, we would have to change it in the future, what do you think?

Reply:
I think its being used in this particular file only. We can continue with this name.

Reply:
Good catch Ajay! I overlooked this. It should stay `channel_routing` to make it consistent with code of [openwisp_controller/geo/channels/routing.py](https://github.com/openwisp/openwisp-controller/blob/b1ed757eded66101a1e1f74a5f3aa71e88fe7810/openwisp_controller/geo/channels/routing.py#L20-L26)

Reply:
> I think its being used in this particular file only. We can continue with this name.

I understand but I generally fear names that are too general. `website`, `application`, `variable`, `flag`, `input` etc.
All are potential pointless dependency related bugs staring us at our faces.

Reply:
> Good catch Ajay! I overlooked this. It should stay channel_routing to make it consistent with code of openwisp_controller/geo/channels/routing.py


Okay, then I will change it. I will need to change the name in this PR too https://github.com/openwisp/openwisp-notifications/pull/156.

Reply:
Thanks @niteshsinha17

Reply:
>  I will need to change the name in this PR too openwisp/openwisp-notifications#156.

No! The following code in openwisp-notifications exists in `tests/openwisp2/routing.py`

```python
application = ProtocolTypeRouter(
        {'websocket': AuthMiddlewareStack(URLRouter(get_routes(consumers)))}
    )
```
Since it lives in the test project, it does not get shipped(and hence, does not matter much). 

The code here in question gets shipped though. So, it is wiser to avoid using such names that could raise conflicts in the future. 

I hope this explanation helps you. :smile:

Reply:
I got your point. Thanks @pandafy